Project/Assets/Prairie/Framework/Script/Interaction/Prompt.cs

```diff
@@ -0,0 +1,20 @@
+using UnityEngine;
+using System.Collections;
+
+public class Prompt : MonoBehaviour 
+{
+
+	public string promptText;
+	
+	public void DrawPrompt()
+	{
+		// Draw a GUI with the interaction 
+		Rect frame = new Rect (Screen.width / 2, Screen.height / 2, Screen.width / 4, Screen.height / 4); 
+
+		GUI.BeginGroup(frame); 
+			GUILayout.Box ("Click to " + promptText); 
```

Comment:
Agreed. 

Is there a way to make the prompt input field start with some default text like "Click to"? That way, the user would get the suggestion that they should put something like "Click to" at the beginning, but they could also delete it if they wanted.